docs/data/material/integrations/nextjs/nextjs.md

```diff
@@ -175,50 +175,55 @@ Inside the `pages/_document.tsx` file:
 - Import `documentGetInitialProps` and use it as the Document's `getInitialProps`.
 - Import `DocumentHeadTags` and render it inside the `<Head>`.
 
-```diff title="pages/_document.tsx"
```

Comment:
@mnajdova `diff` is hard for copy pasting so I changed to `tsx`. Do you think this is a better experience? If yes, I will update the App Router guide too.

Reply:
Between https://deploy-preview-45946--material-ui.netlify.app/material-ui/integrations/nextjs/#configuration-2 and https://deploy-preview-45944--material-ui.netlify.app/material-ui/integrations/nextjs/#configuration-2 It feels like the diff UX wins with a long shot for my use case: I can't copy and paste the whole demo, I have ton of custom logic in my _document.js, but with a diff, I know what to look at.

Reply:
> The best solution is to have CLI that applies the changes, like Shadcn does, but that's a different story #39588 and #40062.

As you said, it's a different story.

For this PR, the change display the smallest snippet to demonstrate the integration. I think it's better than the previous one.